cmd/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/m-kuzmin/daily-reporter/internal/clients/telegram"
+)
+
+func main() {
```

Comment:
How about create configuration file and after that make client setup

Reply:
Thats a good idea!

Reply:
Implemented in https://github.com/m-kuzmin/daily-reporter/pull/2/commits/b74b857d8eefe786e3f843235f2d01f6e0bc65cc